cmd/noms/noms.go

```diff
@@ -44,49 +29,70 @@ var actions = []string{
 	"nomming on",
 }
 
-func usageString() string {
-	i := rand.New(rand.NewSource(time.Now().UnixNano())).Intn(len(actions))
-	return fmt.Sprintf(`Noms is a tool for %s Noms data.`, actions[i])
+// CommandHandler - a callback passed to commands
+type CommandHandler func() (exitCode int)
+
+var (
+	verboseFlag *bool
+	quietFlag   *bool
+)
+
+// AddVerboseFlags adds --verbose and --quiet flags to the passed command
+func AddVerboseFlags(cmd *kingpin.CmdClause) (verboseFlag *bool, quietFlag *bool) {
+	verboseFlag = cmd.Flag("verbose", "show more").Short('v').Bool()
+	quietFlag = cmd.Flag("quiet", "show less").Short('q').Bool()
+	return
+}
+
+// ApplyVerbosity - run when commands are invoked to apply the verbosity arguments configured in AddVerboseFlags
+func ApplyVerbosity() {
+	verbose.SetVerbose(*verboseFlag)
+	verbose.SetQuiet(*quietFlag)
+}
+
+// AddDatabaseArg adds a "database" arg to the passed command
+func AddDatabaseArg(cmd *kingpin.CmdClause) (arg *string) {
+	return cmd.Arg("database", "a noms database path").Required().String() // TODO: custom parser for noms db URL?
+}
+
+// Commands, in order of preference
+var commands = []func(*kingpin.Application) (*kingpin.CmdClause, CommandHandler){
```

Comment:
This line is not formatted correctly. Install `gofmt` and set it to run on save automatically in your editor.